client-app/src/libs/shopping-cart/views/add-to-order-draft-button/add-to-order-draft-button.ts

```diff
@@ -31,15 +36,13 @@ export default class AddToDraftButton extends Vue {
   addItemToDraft!: (criteria: AddCartItem) => void;
 
 
-  async addToNewDraft() {
+  async addToNewDraft(newDraft: AddDraft) {
```

Comment:
can we use "addToDraft(draft: Draft) {...} ? it is shorter

the word "new" it like an extra adjective for this context